api/specs.proto

```diff
@@ -206,14 +206,14 @@ message EndpointSpec {
 		// network. The user may then provide an alternate
 		// option for ingress load balancing which is
 		// transparent to swarm.
-		DISABLED = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
+		LOCAL = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
```

Comment:
`DISABLED` means we won't allocate any `SwarmPort` but the service will still be connected to the ingress network. So a user can launch a HA proxy and expose it as well (so it gets connected to ingress network) and directly load balance to the service using either VIP or DNSRR

#911 is not correct because it tries to bind the `SwarmPort` as a `HostPort` in the container which wouldn't work if there are more than one task of the same service scheduled in a node.